src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -416,3 +417,8 @@ export const day2ApiVipValidationSchema = Yup.string().test(
     return isIPorDN(value);
   },
 );
+
+export const bmcAddressValidationSchema = Yup.string().matches(BMC_REGEX, {
+  message: 'Value "${value}" is not valid BMC address.', // eslint-disable-line no-template-curly-in-string
+  excludeEmptyString: true,
```

Comment:
IIUC this means that empty address is valid which is fine from the address format point of view. However, the BMC address is required field to correctly register bare metal host. So I think we should keep the `.required()` part in the  BMCForm.tsx